app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/functions/setInitialFilters/setInitialFiltersString.ts

```diff
@@ -1,8 +1,8 @@
 import {MapFeatureAttributeState} from "../../../states/map/MapFeatureAttributeState/MapFeatureAttributeState";
 import {setFilter} from "kepler.gl/actions";
-import {MapNumericFeatureAttributeState} from "../../../states/map/MapFeatureAttributeState/MapNumericFeatureAttributeState";
 import {KeplerFilterType} from "../../../states/map/KeplerFilterType";
 import {Dispatch} from "redux";
+import {MapStringFeatureAttributeState} from "../../../states/map/MapFeatureAttributeState/MapStringFeatureAttributeState";
 
 export const setInitialFiltersString = (
```

Comment:
You shouldn't need these freestanding functions, or? This is just the implementation of StringFeatureAttribute.setInitialFilters. If it's not used elsewhere move this code there.